pipeline/dbt/models/intermediate/odspep/int_odspep__services.sql

```diff
@@ -0,0 +1,32 @@
+WITH ressources_partenariales AS (
+    SELECT * FROM {{ ref('int_odspep__enhanced_res_partenariales') }}
+),
+
+final AS (
+    SELECT
+        id_res              AS "id",
+        "service"           AS "nom",
+        'odspep'            AS "source",
+        service_description AS "presentation_resume",
+        date_fin_valid      AS "date_suspension",
+        date_derniere_modif AS "date_modification",
+        code                AS zone_diffusion_code,
+        type_code           AS zone_diffusion_type,
```

Comment:
histoire de distinguer rapidement ce qui releve du nommage de ce qui relève des transformations, et donc histoire de réduire la charge cognitive,
je te propose de faire: **expression** (sans doubles quotes) `AS` "output_name" (avec double quotes)
de cette façon vscode highlight différemment

```suggestion
        service           AS "nom",
        'odspep'            AS "source",
        service_description AS "presentation_resume",
        date_fin_valid      AS "date_suspension",
        date_derniere_modif AS "date_modification",
        code                AS "zone_diffusion_code",
        type_code           AS "zone_diffusion_type",
```

Reply:
rmq: parfois les doubles quotes sont nécessaires si la casse n'est pas bonne, mais ça devrait être nettoyé dans les modèles `sources`